src/core/config/configErrors.ts

```diff
@@ -0,0 +1,52 @@
+import { DSNPError } from "../errors";
+
+/**
+ * ConfigError indicates that some piece of configuration is not set and that
+ * the current action cannot proceed without it.
+ */
+export class ConfigError extends DSNPError {
```

Comment:
I don't see `core/config/configErrors` getting exported out of the sdk, but I think we would want them to be?

Reply:
Fixed